airdrop/airdrop_test.gno

```diff
@@ -292,59 +292,155 @@ func TestClaim64(t *testing.T) {
 }
 
 func TestRefund(t *testing.T) {
-	token := NewMockGRC20("Test Token", "TST", 18)
-	refundTo := testutils.TestAddress("refund_to")
-	config := Config{
-		RefundableTimestamp: uint64(time.Now().Add(time.Hour).Unix()),
-		RefundTo:            refundTo,
-	}
-	airdrop := NewAirdrop(token, config, testutils.TestAddress("airdrop"))
+	mockToken := NewMockGRC20("Test Token", "TST", 18)
+	refundTo := std.Address("refund_to")
```

Comment:
This is not valid address. It was valid long time ago but since this [pr](https://github.com/gnolang/gno/pull/1799) its no longer valid.

Please use testutils to generate address

Reply:
All the usage of `std.Address` has been removed. https://github.com/gnoswap-labs/governance/pull/8/commits/029f047b24d5fdbe956f5fa7d2975ed926435480